From andre@bitwigglers.org  Wed Jan 21 09:13:28 2009
From: Andre Haupt <andre@bitwigglers.org>
Date: Mon, 19 Jan 2009 12:00:17 +0100
Subject: Staging: asus_oled: do not initialise statics to 0 or NULL
To: greg@kroah.com
Cc: Andre Haupt <andre@bitwigglers.org>, sjakub@gmail.com
Message-ID: <9461fb57360808a1d2121f6518cf18318d90aec3.1232362203.git.andre@bitwigglers.org>


fix the following error reported by checkpatch.pl
ERROR: do not initialise statics to 0 or NULL

Signed-off-by: Andre Haupt <andre@bitwigglers.org>
Cc: Jakub Schmidtke <sjakub@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/asus_oled/asus_oled.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/asus_oled/asus_oled.c
+++ b/drivers/staging/asus_oled/asus_oled.c
@@ -56,10 +56,10 @@ MODULE_AUTHOR("Jakub Schmidtke, sjakub@g
 MODULE_DESCRIPTION("Asus OLED Driver v" ASUS_OLED_VERSION);
 MODULE_LICENSE("GPL");
 
-static struct class *oled_class = NULL;
-static int oled_num = 0;
+static struct class *oled_class;
+static int oled_num;
 
-static uint start_off = 0;
+static uint start_off;
 
 module_param(start_off, uint, 0644);
 
